README.md

```diff
@@ -51,13 +51,7 @@ Zero-config PWA Plugin for Nuxt 3
 > For older versions, `@vite-pwa/nuxt` requires Vite 3.2.0+ and Nuxt 3.0.0+.
 
 ```bash
-npm i @vite-pwa/nuxt -D 
-
-# yarn 
-yarn add @vite-pwa/nuxt -D
-
-# pnpm 
-pnpm add @vite-pwa/nuxt -D
+npx nuxi@latest module add vite-pwa-nuxt
```

Comment:
How does nuxi resolve the module name? the name shouldn't be `@vite-pwa/nuxt`?

Reply:
It's coming from the slug in [nuxt/modules](https://github.com/nuxt/modules/). you can also use the npm package name `@vite-pwa/nuxt`